.gitignore

```diff
@@ -63,7 +63,7 @@ typings/
 .node_repl_history
```

Comment:
Overall looks good, will just want a quick look after local build cleaned up and finalized  to merge

Reply:
Not sure if you still want to give it another look or not, since you already approved. I'll merge, but I can do a follow-up PR if necessary.